sopel/bot.py

```diff
@@ -417,6 +421,13 @@ def reply(self, message, destination=None, reply_to=None, notice=False):
                 reply_to = self._trigger.nick
             self._bot.reply(message, destination, reply_to, notice)
 
+        def kick(self, nick, channel=None, message=''):
+            if channel is None:
+                channel = self._trigger.sender
```

Comment:
As I alluded to in my suggested docstring above, if the trigger is in privmsg, not a channel, this should still fall through to a `return` (or throw a nice exception, if we follow @Exirel's way of thinking).